cmd/minikube/cmd/config/profile_list.go

```diff
@@ -101,11 +101,6 @@ var printProfilesTable = func() {
 		}
 
 	}
-
-	if err != nil {
```

Comment:
I would say if there error is not nil we could at least log it, how about glog.Warnningf ?